lwr:
  ft_sensor_calib_controller:
    calib_number_q: 35
    home_q:
      - 0.12783551216125488
      - 0.7029324173927307
      - 0.28488314151763916
      - -1.5327800512313843
      - 1.5623629093170166
      - -0.05475286394357681
      - 1.2095211744308472
    calib_q:
      q12:
        - 1.207030547574698
        - 0.7120867888415852
        - -1.128631830317076
        - -1.473685245508345
        - -1.028084287236641
        - 1.099692883033232
        - 2.921810614489727
      q0:
      - 0.12783551216125488
      - 0.7029324173927307
      - 0.28488314151763916
      - -1.5327800512313843
      - 1.5623629093170166
      - -0.05475286394357681
      - 1.2095211744308472

      q9:
        - 1.117063772347047
        - 0.6195448709862337
        - -0.9356569418360889
        - -1.473584070052457
        - -1.171736632549334
        - 1.200034882640741
        - 1.652275521743421
      q20:
        - 1.291924455964918
        - 1.344446616602421
        - -1.835025699132549
        - -1.622942684889253
        - -1.25354156747679
        - 0.3270112604920214
        - -2.838316749378947
      q21:
        - 1.275097875943948
        - 1.319340558331526
        - -1.7750104055809
        - -1.575161760185878
        - -1.136385706447335
        - 0.6313500387861328
        - -2.822149328682925
      q22:
        - 1.240292814674214
        - 1.332986795683989
        - -1.769527461511012
        - -1.509834991508731
        - -0.9351590705036479
        - 0.8754400749421558
        - -2.662755327003574
      q23:
        - 1.204567025148164
        - 1.334094102864507
        - -1.771595186007473
        - -1.437219079553305
        - -0.7382404179177637
        - 1.101509155686552
        - -2.482541370263808
      q1:
        - 0.4578696699247446
        - 0.3350588038054285
        - -0.1470711626720957
        - -1.464278450379614
        - 0.190213116364415
        - 0.8488940655112831
        - 1.288512142045775
      q24:
        - 0.8625319329400014
        - 0.8995364679315676
        - -1.106264084384027
        - -1.153339311790409
        - -0.500632484726979
        - 2.763807239504015
        - 2.507202118353474
      q25:
        - 0.9368251807128409
        - 0.9541468525731567
        - -1.219621946770582
        - -1.140003151667997
        - -0.6014114663741914
        - 2.381610071455043
        - 2.51634310861138
      q26:
        - 1.025362084850056
        - 0.983584370769564
        - -1.301569751738103
        - -1.185399880855755
        - -0.6406390091949126
        - 1.982643899560681
        - 2.651816533536802
      q27:
        - 1.099194291994068
        - 1.040057926092813
        - -1.395731627719395
        - -1.258544486811683
        - -0.7676396396705361
        - 1.860698626935026
        - 2.852996016792307
      q28:
        - 1.217358788975613
        - 1.00633301910443
        - -1.414437706366243
        - -1.408932562862931
        - -0.9414944850433242
        - 1.329453488413471
        - 2.953510973210498
      q29:
        - 1.282742776889377
        - 1.117291722720606
        - -1.565857874881446
        - -1.52786539291569
        - -1.173818332315127
        - 0.9301893514019772
        - -2.769278793072891
      q30:
        - 1.286662697199208
        - 1.249557348011507
        - -1.700252727049615
        - -1.593521341230181
        - -1.420001382158449
        - 0.7064184489345084
        - -2.350816916181495
      q31:
        - 1.301855847365768
        - 0.5923514549930564
        - -1.275908212999854
        - -1.633862286438649
        - -2.238251472741386
        - -0.1242175915400665
        - 2.880501683832376
      q17:
        - 1.322116988649505
        - 1.219241096104044
        - -1.855757108642262
        - -1.609333445810464
        - -1.953080443973142
        - -0.4160575814451279
        - 2.922979092692723
      q18:
        - 1.325415016075421
        - 1.255417379071753
        - -1.849408409387563
        - -1.639759741884319
        - -1.769461183208042
        - -0.1754336300673547
        - 2.892053895536542
      q19:
        - 1.320570406072801
        - 1.279489429301853
        - -1.821783693031135
        - -1.645812822703157
        - -1.494794791319075
        - 0.10003160681857
        - 2.955771259857935
      q32:
        - 1.150010402737348
        - 1.641759851042591
        - -2.195713128841239
        - -1.622662095463727
        - -1.849816606819381
        - -0.3507324699087855
        - 2.919715111551156
      q33:
        - 0.9184080322354227
        - 1.892414935398918
        - -2.520897461476378
        - -1.607099713122817
        - -1.99647148228087
        - -0.4121990627558256
        - -2.812953495469726
      q34:
        - 0.6676830648811105
        - 2.025887605135015
        - -2.827135571004433
        - -1.587541104450422
        - -2.258593938034045
        - -0.439617052862916
        - -2.764212513572379
      q5:
        - 1.130377155978582
        - 0.7346945392363962
        - -1.029608178459545
        - -1.454798034787319
        - -1.336981584689511
        - 1.421561086430787
        - 2.467074309188728
      q14:
        - 1.25975602657245
        - 0.8062827708587124
        - -1.280948833223602
        - -1.474028904778393
        - -0.8831648498383933
        - 1.01383341038863
        - -2.465123587874244
      q15:
        - 1.282645015557399
        - 1.169111907395432
        - -1.842651007837672
        - -1.499883399711949
        - -2.304857788339835
        - -0.8737730122703837
        - 2.638065955268373
      q16:
        - 1.300349018901355
        - 1.183630212641989
        - -1.852338738622665
        - -1.542923159457075
        - -2.185562062433917
        - -0.7198978796853304
        - 2.742853845035848
      q4:
        - 1.096335704052854
        - 0.6039015404530543
        - -0.8966419688265255
        - -1.47343224450676
        - -1.194775338581724
        - 1.218229158460186
        - 2.388360580445923
      q11:
        - 1.189433181412083
        - 0.6895583854021075
        - -1.086689590421098
        - -1.473559532960577
        - -1.062342010457375
        - 1.122630126208006
        - 2.810705899579565
      q13:
        - 1.239877335859367
        - 0.7644097234046434
        - -1.217063821289868
        - -1.473841830154118
        - -0.9480429774811601
        - 1.050371963683291
        - -2.82325902637738
      q7:
        - 1.072928047704162
        - 1.119312715078287
        - -1.425403446234752
        - -1.389052930250644
        - -1.641791285988968
        - 1.97503791624467
        - 2.457963326888811
      q6:
        - 1.131039862016884
        - 0.8703325386966885
        - -1.15886151664494
        - -1.431996735545073
        - -1.446227798763565
        - 1.620746920134254
        - 2.498676102828551
      q8:
        - 1.089340211165994
        - 0.5990580141178095
        - -0.8836339607432224
        - -1.473843888846107
        - -1.205252104878211
        - 1.224369121988236
        - 1.163629938672253
      q10:
        - 1.133881986064716
        - 0.6333583638870044
        - -0.9684705300375924
        - -1.473528894722143
        - -1.149708684336053
        - 1.183949017212997
        - 1.944553274746223
      q2:
        - 0.8770416471497571
        - 0.3996730978934462
        - -0.57394180183993
        - -1.487632958986267
        - -0.6745307951622337
        - 0.8619363100585549
        - 1.995800373815415
      q3:
        - 1.017009201484492
        - 0.4889124425667104
        - -0.7504973178569525
        - -1.485060244196266
        - -0.9874718757448608
        - 1.021968962825922
        - 2.242546977691962